### PR TITLE
[MetaSchedule] Handle deserializing empty string RVs in trace

### PR DESCRIPTION
**Pull request by AndrewZhaoLuo**
It seems sometimes we can have random variables with names such as "" (this is not the empty string, but two quotation marks). 

I believe this is suppose to hit "Case 2" in `TranslateInputRVs` and become an empty string but this does not seem to be the case.

Unsure if having an RV as "" may be problematic, if it is, this will certainly hide another bug

cc @junrushao1994

### CODE REVIEW

File: tests/python/unittest/test_tir_schedule_trace.py [hunk 1]
```diff
@@ -275,5 +284,53 @@ def test_apply_json_to_schedule_1():
...
+    _test_apply_annotation_trace_from_json("SSRSSR")
+
+    # The empty string
+    _test_apply_annotation_trace_from_json("")
+
+    # A string of two quotation marks
+    _test_apply_annotation_trace_from_json('""')
+
+    # A string of one quotation mark
+    _test_apply_annotation_trace_from_json('"')
```
Comment (zxybazh) [inline tests/python/unittest/test_tir_schedule_trace.py:333]: Nit picking, I find this quotation mark evil, because it's not matched and any single character would be good enough for this test case right?

**Review by zxybazh — APPROVED** [review]
LGTM, only nit picking this evil quotation mark.

File: tests/python/unittest/test_tir_schedule_trace.py (hunk 1 above)
Comment (AndrewZhaoLuo) [reply tests/python/unittest/test_tir_schedule_trace.py:333]: It's supposed to be evil :^). 

Any single other character would work but it's still an edge case from a black box point of unittesting.

Comment (junrushao1994) [reply tests/python/unittest/test_tir_schedule_trace.py:334]: super evil. i love that!
